.github/workflows/python.yml

```diff
@@ -31,6 +31,12 @@ jobs:
       run: |
         sudo apt update
 
+    - uses: Chocobo1/setup-ccache-action@v1
```

Comment:
what happens on windows? ccache doesn't support windows, right?

Reply:
Right, it's a no-op.